foundation/foundation-configuration-store/README.md

```diff
@@ -21,4 +21,7 @@ properties:
 
 Because the `YamlProperty` and `YamlConfig` classes are required to have attributes that mirror the format of the configuration `YAML` files, there is a separate `Property` class that simply adds the `groupName` metadata to the existing information housed in `YamlProperty`. Furthermore, because an object that will house the deserialized `YAML` contents must have an empty constructor, it was not possible to have any extending relationship between a supposed `AbstractProperty` and `YamlProperty` / `Property`.
 
-See the [snakeyaml documentation](https://bitbucket.org/snakeyaml/snakeyaml/wiki/Documentation) for further details on deserializing yaml files.
\ No newline at end of file
+See the [snakeyaml documentation](https://bitbucket.org/snakeyaml/snakeyaml/wiki/Documentation) for further details on deserializing yaml files.
+
+### Handling Load Status
+The `ConfigLoader` can leverages `load-status` to detect whether properties were already fully loaded or only partially loaded due to an interrupted process. The code logic ensures that a `fully-loaded` status is set when properties are loaded successfully. This status is then used to skip loading if the properties were successfully loaded previously, thus avoiding unnecessary reloading during deployment refreshes. For details on this behavior, refer to the implementation code and comments in the `ConfigLoader` class.
```

Comment:
`The `ConfigLoader` can leverages `load-status`` -> `The `ConfigLoader` leverages `load-status``

Reply:
fixed it